README.md

```diff
@@ -52,4 +50,53 @@ If you are facing a path too long exception when building this example project,
+
+### Alignment 
+Align badges to their content in start, end, and center modes.
+
+### Icon
+The .NET MAUI Badge View control provides predefined notification symbols such as available, busy, away, delete, and more.
+
+### Customization
+Customize the MAUI Badge View control’s background color, text color, and font styles easily.
+
+## Related links
+[Learn More about .NET MAUI Badge View](https://www.syncfusion.com/maui-controls/maui-badge-view?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples)
+
+[Documentation](https://help.syncfusion.com/maui/badge-view/getting-started?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-badge-view-control?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples)
+
+[Online example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/BadgeView/SampleBrowser.Maui.BadgeView/Samples/BadgeView?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples)
+
+## About Syncfusion .NET MAUI Controls
+
+Syncfusion's [.NET MAUI UI Controls](https://www.syncfusion.com/maui-controls?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples) library is the only suite that you will ever need to build an application since it contains over 40 high-performance, lightweight, modular, and responsive UI Controls in a single package. In addition to Radial Gauge, we provide popular .NET MAUI Controls such as [DataGrid](https://www.syncfusion.com/maui-controls/maui-datagrid?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples)[Charts](https://www.syncfusion.com/maui-controls/maui-cartesian-charts?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples), [Scheduler](https://www.syncfusion.com/maui-controls/maui-scheduler?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples), [ListView](https://www.syncfusion.com/maui-controls/maui-listview?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples), and [Excel Library](https://www.syncfusion.com/document-processing/excel-framework/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-badge-view-github-samples).
+
```

Comment:
Update the correct control here:
![image](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-badge-view/assets/95272441/56600c96-a5bd-436e-8090-00b6eaf50c72)

Reply:
Updated the correct control.